pyathena/sqlalchemy_athena.py

```diff
@@ -31,6 +31,34 @@
 
 import pyathena
 
+BLANKS = re.compile(r"\s+")
+LIMIT_COMMENT_COLUMN = 255
+# https://docs.aws.amazon.com/athena/latest/APIReference/API_Column.html
+LIMIT_COMMENT_TABLE = 51200
+# https://docs.aws.amazon.com/athena/latest/APIReference/API_TableMetadata.html
```

Comment:
If this limit is exceeded, does Athena return an error or is it truncated? There is no need to manage this condition in dialect.

Reply:
I don't know for tables, did not manage to reach the limit, though the documentation mentions this one.
For columns you do get en error:
```
CREATE EXTERNAL TABLE default.plop(
   number INT COMMENT 'abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789'
)
STORED AS parquet
LOCATION 's3://jt-eu-west-1-dev-datalake-pyathena-testing-bucket/nicolas/default/plop';
```
They will fail with:
```
FAILED: Execution Error, return code 1 from org.apache.hadoop.hive.ql.exec.DDLTask. MetaException(message:1 validation error detected: Value 'abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789abcdefghij0123456789' at 'table.storageDescriptor.columns.1.member.comment' failed to satisfy constraint: Member must have length less than or equal to 255
```

Reply:
Please do not do any truncate processing here.
It is better to raise the Athena error as it is.